documentation/assemblies/upgrading/assembly-upgrade-cluster-operator.adoc

```diff
@@ -33,12 +33,6 @@ The `helm upgrade` command does not upgrade the {HelmCustomResourceDefinitions}.
 Install the new CRDs manually after upgrading the Cluster Operator.
 You can access the CRDs from the {ReleaseDownload} or find them in the `crd` subdirectory inside the Helm Chart.
 
-== Migrating to unidirectional topic management
-
-When deploying the Topic Operator to manage topics, the Cluster Operator enables unidirectional topic management by default. 
-If you are switching from a version of Strimzi that used bidirectional topic management, there are some cleanup tasks to perform after upgrading the Cluster Operator.
-For more information, see xref:proc-changing-topic-operator-mode-{context}[].
-
```

Comment:
I think we should keep this. This still applies to upgrades from older versions. Maybe we should just rename the title?

Reply:
Yeah. Now I remember why I didn't remove it. But we should remove the dead link, and maybe update the title as suggested by Jakub.

Reply:
The procedure we link to came out with the PR to promote the UTO: https://github.com/strimzi/strimzi-kafka-operator/pull/9840
Do we need to reinstate it?

Reply:
Maybe you can simply inline here that they should delete the old `__strimzi...` topics.